src/screens/SendScreen.tsx

```diff
@@ -122,10 +127,7 @@ const SendScreen = ({
     convertAlphToSet(value) >= MINIMAL_GAS_PRICE ||
     `Gas price must be at least ${formatAmountForDisplay(MINIMAL_GAS_PRICE, true)}`
 
-  useEffect(() => {
-    if (txStep === 'send') setTxStep('build')
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [fromAddress, toAddressHash, amountInAlph, gasAmount, gasPriceInAlph])
```

Comment:
@mvaivre Thanks for making me more conscious about my use of disabling the eslint rule. I figured I can simplify the code since `setState` [does not cause a re-render when the value didn't change](https://reactjs.org/docs/hooks-reference.html#bailing-out-of-a-state-update). Now, there's no need to disable the rule.

Reply:
Awesome 🎉